contact_info_service.py

```
@@ -1,19 +1,23 @@
 import json
 from typing import ClassVar
+from telemetry import tracer
+from opentelemetry.trace import Status, StatusCode
 
 
 class ContactInfoServiceError(Exception):
-    """Exception raised when ContactInfoService initialization or operation fails."""
+    """Exception raised when ContactInfoService initialization or operation
+    fails."""
 
     pass
```

Ruff (0.14.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)